types/error-cause/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for error-cause 1.0
+// Project: https://github.com/es-shims/error-cause
+// Definitions by: Conrad Buck <https://github.com/conartist6>
```

Comment:
```suggestion
// Definitions by: Conrad Buck <https://github.com/conartist6>
// Definitions by: Jordan Harband <https://github.com/ljharb>
```

Reply:
I mean, really? I don't wanna be pedantic, but this was a lot of work and I'm the one who did it.

Reply:
unfortunately it's the only way DefinitelyTyped will auto-tag me on future PRs to the types, or let me approve them on a review - it's not about claiming credit.

Reply:
I guess it's what should be there, as that way any future editors will get both of us as reviewers, which is what they should get.